.changes/next-release/enhancement-Eventstream-60668.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "Eventstream",
+  "description": "The limit for event streams maximum payload size was raised from 16Mb to 24Mb."
```

nit; we weren't enforcing 16Mb prior to this release, we should try to frame this as the effective change.
```suggestion
  "description": "The event streams maximum payload size is now required to be 24Mb or less."
```